tools/analyzer_plugin/lib/src/diagnostic/duplicate_prop_cascade.dart

```diff
@@ -27,11 +27,13 @@ class DuplicatePropCascadeDiagnostic extends ComponentUsageDiagnosticContributor
   @override
   computeErrorsForUsage(result, collector, usage) async {
     final propUsagesByName = groupBy<PropAssignment, String>(usage.cascadedProps, (prop) {
+      final enclosingElement = prop.name.staticElement?.enclosingElement?.name ?? '';
+
       if (prop.targetName != null && prop.targetName.name != 'dom') {
-        return '${prop.targetName.name}.${prop.name.name}';
+        return '$enclosingElement.${prop.targetName.name}.${prop.name.name}';
       }
```

Comment:
Do we need this check now, or can we remove this whole if statement?

Reply:
I'm pretty sure we still need it - the `targetName` is different than the `enclosingElement`.

Without this check - the following props would be flagged as dupe since they both have an `enclosingElement` name of `AriaPropsMixin`:

```dart
(Dom.div()
  ..hidden = true
  ..aria.hidden = true
)()
```

Reply:
> Without this check - the following props would be flagged as dupe since they both have an enclosingElement name of AriaPropsMixin

Wouldn't we _want_ that case flagged as a dupe though?

Reply:
No - they are different.

One results in

```html
<div hidden></div>
```

The other results in 

```html
<div aria-hidden="true"></div>
```

Reply:
Oh wait... I think I see what you're saying now - hmmm... maybe we don't need it?